policyengine_us/variables/gov/states/sc/tax/income/exemptions/sc_military_retirement_income_deduction.py

```diff
@@ -0,0 +1,10 @@
+from policyengine_us.model_api import *
+
+
+class sc_military_retirement_income_deduction(Variable):
```

Comment:
Same for retirement income
```suggestion
class sc_military_retirement_income_deduction_head(Variable):
```